gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1825,6 +1834,68 @@ protected GenerateDataTemplateTask configureDataTemplateGeneration(Project proje
     // FIXME change to #getArchiveFile(); breaks backwards-compatibility before 5.1
     project.getDependencies().add(compileConfigName, project.files(dataTemplateJarTask.getArchivePath()));
 
+    project.getPlugins().withType(IvyPublishPlugin.class, ivyPublish -> {
```

Comment:
You already have a lot of documentation which is good, but it would be a good idea IMO to leave a comment here to describe at a high level what this closure is for. The plugin is already hard enough for us to understand, we need all the documentation we can get.

Reply:
Sure, I'll add extra docs.